docs/src/main/asciidoc/opentelemetry-dev-services.adoc

```diff
@@ -0,0 +1,41 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Dev Services for OpenTelemetry
+:extension-status: preview
+include::_attributes.adoc[]
+:categories: data
+:summary: Start OpenTelemetry automatically in dev and test modes.
+
+Quarkus supports a feature called Dev Services that allows you to run collect traces without any config.
+What that means practically, is that if you have docker running and have not configured `quarkus.otel.exporter.otlp.traces.endpoint`,
+Quarkus will automatically start a Jaeger container when running tests or dev mode, and automatically configure the connection.
+
+Available properties to customize the OpenTelemetry DevService.
+
+include::{generated-dir}/config/quarkus-opentelemetry-config-group-devservices-dev-services-config.adoc[opts=optional, leveloffset=+1]
+
+When running the production version of the application, the exporter connection needs to be configured as normal,
+so if you want to include a production tracing config in your `application.properties` and continue to use Dev Services
+we recommend that you use the `%prod.` profile to define your exporter configuration settings.
+
+Dev Services for OpenTelemetry relies on Docker to start the server.
+If your environment does not support Docker, you will need to start the server manually, or connect to an already running server.
+
+NOTE: If you want to disable ryuk container, set variable `TESTCONTAINERS_RYUK_DISABLED=true`.
```

Comment:
This seems out of context and if we disable RYUK and kill app in dev mode, container will keep running, isn't it? Can you add context (to docs, not to comment) why you write it there, please?